stl/inc/xutility

```diff
@@ -5988,49 +5968,40 @@ namespace ranges {
                     }
                 }
 
-                using _Ptr_t = remove_reference_t<_Iter_ref_t<_It>>*;
-#if _USE_STD_VECTOR_ALGORITHMS
+                using _Ptr_t          = remove_reference_t<_Iter_ref_t<_It>>*;
                 const auto _First_ptr = _STD _To_address(_First);
 
                 _Ptr_t _Result;
-
-                if constexpr (_Is_sized) {
+#if _USE_STD_VECTOR_ALGORITHMS
+                if constexpr (!_Single_byte_elements) {

```

Comment:
```suggestion
                if constexpr (!_Single_byte_elements || _Is_sized) {
```

We want `__std_find_trivial` for sized single byte element, it is faster than `memchr`